Automattic/vip-go-mu-plugins
# Fix ESLint issues in Search Dev Tools

## Description

Somewhere along the way eslint got borked via one of renovatebot merges, this should address.

## Changelog Description

### Internal coding standards improvements

We've fixed ESLint configs in one of our mu-plugins

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1.  Merge and `cd  search/search-dev-tools`
2.  `npm run lint` and ensure ESLint runs successfully.
